meta-mentor-staging/security/recipes-ids/tripwire/tripwire_%.bbappend

```diff
@@ -4,7 +4,7 @@ RDEPENDS_${PN}_remove = "${RDEPENDS_REMOVE}"
 
 do_install_append () {
     # When submitting upstream, modify twcfg.txt in the layer instead
-    sed -i -e 's#^EDITOR=#EDITOR=/usr/bin/vi#' ${D}${sysconfdir}/${PN}/twcfg.txt
+    sed -i -e 's#^EDITOR[[:space:]]*=.*#EDITOR=/usr/bin/vi#' ${D}${sysconfdir}/${PN}/twcfg.txt
```

Comment:
for some reason, this build error didn't appear while building and testing security-utils, see runtime logs
https://jira.alm.mentorg.com/browse/SB-15276
now it does appear and needs to be fixed. Shouldn't affect the functionality of the utility